waffle-chai/test/matchers/events.test.ts

```diff
@@ -39,6 +41,15 @@ describe('INTEGRATION: Events', () => {
     );
   });
 
+  it('Emit index: success', async () => {
+    const bytes = ethers.utils.hexlify(ethers.utils.toUtf8Bytes('Three'));
+    const hash = ethers.utils.keccak256(ethers.utils.toUtf8Bytes('Three'));
+    await expect(
```

Comment:
Please format as:
```ts
await expect(events.emitIndex())
  .to.emit(events, 'Index')
  .withArgs(
    hash,
    'Three',
    bytes,
    hash,
   '0x00cfbbaf7ddb3a1476767101c12a0162e241fbad2a0162e2410cfbbaf7162123'
  )
```

Reply:
Wouldn't it make more sense for the expectation to look like this?
Under the hood the matcher would hash the required arguments. In this way the user doesn't have to figure out the hashing themselves. Alternatively we can support both - if a hash is provided it is checked directly, otherwise a hash is computed on the fly.
```ts
  .withArgs(
    "Three",
    "Three",
    bytes,
    bytes,
    '0x00cFBbaF7DDB3a1476767101c12a0162e241fbAD2a0162e2410cFBbaF7162123'
);
```

Reply:
yes as you can see in the commit that is more or less what i did in the previous commit I will update it. What prettier conf / eslint rules are you using exactly ?

Reply:
@sz-piotr done